src/main/scala/amba/ahb/Protocol.scala

```diff
@@ -37,8 +37,8 @@ object AHBParameters
   def RESP_SPLIT = UInt(3, width=2)
 
   def PROT_DATA        = UInt(1, width = protBits)
-  def PROT_PRIVILEDGED = UInt(2, width = protBits)
+  def PROT_PRIVILEGED  = UInt(2, width = protBits)
   def PROT_BUFFERABLE  = UInt(4, width = protBits)
   def PROT_CACHEABLE   = UInt(8, width = protBits)
-  def PROT_DEFAULT = PROT_DATA | PROT_PRIVILEDGED
+  def PROT_DEFAULT = PROT_DATA | PROT_PRIVILEGED
```

Comment:
Thanks for this typo fix:
However for the happiness of RocketChip downstream, would you mind adding a function alias to not break the rocket-chip downstream?:
```
@deprecated(xxx)
def PROT_PRIVILEDGED = PROT_PRIVILEGED
```

Reply:
I remember in the last dev meeting, we decided not maintain API stability so aggressively.